Source/Core/Core/TimePlayed.cpp

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <string>
+
+#include "Common/IniFile.h"
+#include "Common/FileUtil.h"
+#include "TimePlayed.h"
+
+TimePlayed::TimePlayed(std::string game_id)
+{
+  m_game_id = game_id;
+}
+
+void TimePlayed::AddTime(u64 time_emulated)
+{
+  IniFile ini;
+  std::string ini_path = File::GetUserPath(D_CONFIG_IDX) + "TimePlayed.ini";
+  ini.Load(ini_path);
+
+  auto time_list = ini.GetOrCreateSection("Time Played");
+  int previous_time = 0;
+
+  time_list->Get(m_game_id, &previous_time);
+  time_list->Set(m_game_id, previous_time + int(time_emulated)); //unlikely u64 to int conversion issue
```

Comment:
Is there a reason you're not just storing an `u64` in the ini?

Reply:
The INI set method requires an integer so I have to convert it beforehand.

Reply:
Yeah I fixed that, I didn't realize that the set method was templated.

Reply:
The issue is that it stores a hex value in the TimePlayed.ini file. While this doesn't have any issues in the QT GUI, it does make it harder to parse the milliseconds when looking at the file directly.